lib/active_snapshot/models/concerns/diffable.rb

```diff
@@ -0,0 +1,95 @@
+module ActiveSnapshot
+  module Diffable
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def diff(from:, to:)
```

Comment:
I had a few more thoughts here
1. `from` should always be a snapshot, since it will always be newer
2. the `from` snapshot should always be older than the `to` snapshot otherwise the changes will not make sense.

Do you agree?